### ISSUE TITLE
EASE_IN functions break react-scripts build

### ISSUE THREAD

**Issue by kkwteh** [josepot/react-redux-scroll#13]
There is a problem with this package when I try to minify it for a production build.

```
npm run-script build

> augranger@0.0.1 build /Users/kevinteh/dev/augranger
> react-scripts build

Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file:

 	./node_modules/react-redux-scroll/es/scroll.js:19
```

The line in question is `return t ** 3`. When I comment out the EASE_IN functions with the `**` operator from the code and then re-run the build script, the code minifies successfully.

**Comment by josepot** [comment]
Hi @kkwteh and thanks for reporting this bug and for using this library!

I'm working on a new version of this library and that new version should also solve this issue. I will try to have it finished and published before the end of this week. 

In the meanwhile, using the [following babel plugin](https://babeljs.io/docs/en/babel-plugin-transform-exponentiation-operator) in your build process should solve the issue.

**Comment by josepot** [comment]
Hi @kkwteh ,

As I mentioned in my previous comment I'm working on a new major version. However, I do not want that to delay this bug fix, so I just published version `0.1.2` which fixes this issue. Please let me know if that works for you and thanks for using this library.

**Comment by kkwteh** [comment]
Hi @josepot,
 
Thanks for the reply and the fix. Everything is working now. thanks!
